# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/praenti/ddd/ddd-3.3.1-gcc3-streampos.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ddd-3.3.1-org/ddd/streampos.h	2000-01-17 15:53:42.000000000 +0100
+++ ddd-3.3.1/ddd/streampos.h	2003-02-20 23:04:59.000000000 +0100
@@ -39,7 +39,7 @@
 
 #include "config.h"
 
-#if !HAVE_STREAMPOS
+#if (!HAVE_STREAMPOS) && (!__GNUC__ >= 3 ) && (!__GNUC_MINOR__ >= 2)
 typedef long streampos;
 #endif
 
